pkg/proxy/endpointslicecache.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package proxy
+
+import (
+	"sort"
+
+	discovery "k8s.io/api/discovery/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/klog"
+	utilproxy "k8s.io/kubernetes/pkg/proxy/util"
+	utilnet "k8s.io/utils/net"
```

Comment:
Same thoughts as above, in this case calls importing as `net` seems like it could be confused with the stdlib `net`, but I'm still pretty new to Go and not sure what's standard practice here, was just going off of what I saw elsewhere in kube-proxy.

Reply:
A quick search of the k8s codebase for "k8s.io/utils/net" shows that it's always required with some kind of name, most commonly `utilnet`. I can't find any cases of it being used as just `net`.